backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/Mailer.kt

```diff
@@ -148,7 +148,7 @@ object Mailer {
             projectConfig.smtp,
             projectConfig.administrationName,
             applicationVerification.contactEmailAddress,
-            "Bestätigung notwendig: Antrag auf Bayerische Ehrenamtskarte",
+            "Bestätigung notwendig: Antrag auf Bayerische Ehrenamtskarte [${applicationVerification.contactName}]",
```

Comment:
I guess we don't want to send the contact name but the applicant name ?

Reply:
Jep you're right. Once again a splendid review. Will change that.